### ISSUE TITLE
finetuning pretrained model

### ISSUE THREAD

**Issue by sravani40** [espnet/espnet#3918]
i  have one pretrained model trained on female voice, i want to finetune it to male voice. shall i follow the regular procedure of finetuning to do like that?

**Comment by kan-bayashi** [comment]
Almost the same.
1. Make recipe.
2. Run the recipe with some options.

You can refer jvs recipe as an example.
https://github.com/espnet/espnet/blob/master/egs2/jvs/tts1/README.md

**Comment by sravani40** [comment]
i am trying to finetune on 2 hours of data. original model is trained on 23 hours of data. in the fast speech original learning rate is 1,num_iters_per_epoch: 600  # number of iterations per epoch
max_epoch: 1000           # number of epochs

how much changes in these parameters might do better?

do i need to change any other parameters?

**Comment by kan-bayashi** [comment]
Please check.
https://github.com/espnet/espnet/tree/master/egs2/jvs/tts1/conf/tuning
Basically, not so much iterations are not needed and use smaller learning rate.
